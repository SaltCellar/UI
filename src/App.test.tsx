import { shallow } from 'enzyme';
import React from 'react';
import App from './App';

// TODO: Replace with useful test
test('renders', () => {
  const view = shallow(<App locale="en" />);
  expect(view).toMatchSnapshot();
});
