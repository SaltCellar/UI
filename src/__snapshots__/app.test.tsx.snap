// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders login if isLoggedIn is false 1`] = `
<I18nProvider
  locale="en"
>
  <Async />
</I18nProvider>
`;

exports[`renders page layout and routes if isLoggedIn is true 1`] = `
<I18nProvider
  locale="en"
>
  <Connect(PageBase)
    masthead={<Connect(MastheadBase) />}
    sidebar={<Connect(SidebarBase) />}
  >
    <Routes />
  </Connect(PageBase)>
</I18nProvider>
`;
